apple-pie/styles/style.css

```diff
@@ -0,0 +1,72 @@
+h1 {
+  color: white;
+  background-image: url(/apple-pie/images/apple-pie.jpg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100%;
+  height: 250px;
+  text-align: center;
+  padding-top: 200px;
+  font-size: 70px;
+}
+
+p {
+  font-size: 22px;
+  text-align: center;
+  width: 50%;
+  margin-left: 500px;
+  margin-right: -300px;
```

Comment:
Here you are relying on the margin left and right to center your header paragraph (and same for the image "recipe-info"), so it looks good on a big screen but not so much on smaller screens. 
You could place those 2 elements in a < div > for example and apply flexbox on it in order to center them in a more reliable way :)